cli/tests/pcluster/config/test_validators.py

```diff
@@ -937,3 +937,21 @@ def test_ebs_settings_validator(mocker, cluster_section_dict, ebs_section_dict,
 def test_shared_dir_validator(mocker, section_dict, expected_message):
     config_parser_dict = {"cluster default": section_dict}
     utils.assert_param_validator(mocker, config_parser_dict, expected_message)
+
+
+@pytest.mark.parametrize(
+    "base_os, expected_message",
+    [
+        ("alinux", "Please double check the 'base_os' configuration parameter"),
```

Comment:
What is this checking for?

Reply:
It's checking that when using dcv with OSs that are not supported we are printing out error message. Full error message is `"NICE DCV can be used with one of the following operating systems: ["centos7", "ubuntu1804"]. Please double check the 'base_os' configuration parameter"`